src/main/blackbox/blackbox.c

```diff
@@ -358,7 +358,7 @@ typedef struct blackboxMainState_s {
     int16_t gyroUnfilt[XYZ_AXIS_COUNT];
 #ifdef USE_ACC
     int16_t accADC[XYZ_AXIS_COUNT];
-    int16_t attitude[XYZ_AXIS_COUNT];
+    int16_t imuAttitudeQuaternion[XYZ_AXIS_COUNT];
```

Comment:
(XYZ_AXIS_COUNT stays here, even when whole imuAttitudeQuaternion is exported from imu)